### PR TITLE
OSDOCS-8176: adds release note for MicroShift 4.13.17

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13

Issue:
https://issues.redhat.com/browse/OSDOCS-8176

Link to docs preview:
https://66357--docspreview.netlify.app/microshift/latest/microshift_release_notes/microshift-4-13-release-notes#microshift-4-13-17-dp

QE review:
- N/A standard asynch release note, no new bugs added

### CODE REVIEW

File: microshift_release_notes/microshift-4-13-release-notes.adoc [hunk 1]
```diff
@@ -290,4 +290,15 @@ Issued: 2023-10-10
...
+For the `TopoLVM` image, see link:https://catalog.redhat.com/software/containers/lvms4/topolvm-rhel9/63972de3adcb55263891b983?container-tabs=dockerfile[lvms4/topolvm-rhel9].
+
+//release 4-13-16 was folded into 4-13-17
+
+[id="microshift-4-13-17-dp"]
+=== RHBA-2023:5674 - {product-title} 4.13.17 bug fix update
+
+Issued: 2023-10-17
+
+{product-title} release 4.13.17 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5469[RHBA-2023:5674] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5467[RHBA-2023:5672] advisory.
```
Comment (kelbrown20) [inline microshift_release_notes/microshift-4-13-release-notes.adoc:302]: Just need to update the text part.
```suggestion
{product-title} release 4.13.17 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:5474[RHBA-2023:5674] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5472[RHBA-2023:5672] advisory.
```

**Review by kelbrown20 — COMMENTED** [review]
Hey Shauna, just a nit that needs updating. Otherwise, this LGTM!

**Review by opayne1 — APPROVED** [review]

**Pull request merged**